mcx/sample.py

```diff
@@ -127,7 +127,6 @@ def __init__(
 
         print("sampler: build and compile the inference kernel")
         kernel_factory = evaluator.kernel_factory(loglikelihood)
-        kernel_factory = jax.jit(kernel_factory, static_argnums=(0, 1, 2))
```

Comment:
I was going to pass the kernel factory as an argument, as I was sure it wasn't jitted :D I'm really glad you had a look at it.